opengever/maintenance/scripts/repository_migration.py

```diff
@@ -24,8 +24,6 @@
 transaction.commit()
 
 Notes:
-- Permissions are only taken into account if some local_roles are set (i.e. it
-  is currently not possible to set local_roles back to None)
```

Comment:
It was not implemented because back then, I did not expect that the migration excel would be complete / up to date with the state of the productive system, so I did not want to overwrite permissions that were added by hand. It turns out, that the excels are up to date and they expect the state after the migration to reflect the data in the migration excel.